stdlib/prelude.mt

```diff
@@ -19,3 +19,4 @@ require "./map.mt"
 require "./range.mt"
 require "./string.mt"
 require "./time.mt"
+require "./env.mt"
```

Comment:
keeping these in alphabetic order would be nice.

Reply:
Fixed.

(Wondering whether this will make github understand this is taken care of...)

Reply:
doesn't look like it's been edited...

Reply:
What do you mean?

Reply:
The diff doesn't show that this moved, and it's still at the end on master now: https://github.com/myst-lang/myst/blob/546c2877be8b58a668bd724a936a55c2c80e0308/stdlib/prelude.mt#L22

Not a big deal, though.